nixos/doc/manual/release-notes/rl-2205.section.md

```diff
@@ -6,6 +6,10 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 ## Highlights {#sec-release-22.05-highlights}
 
+- `ngrok` has been upgraded from 2.3.40 to 3.0.2. Please see [the upgrade guide](https://ngrok.com/docs/guides/upgrade-v2-v3)
```

Comment:
This isn't a highlight. This should be either under `Backward Incompatibilities` or `Other Notable Changes`.

Reply:
Oh I missed that section. Moved it and fixed a typo. Thanks!

Reply:
Since 22.05 is already released now, this should probably be moved to the 2211 release notes. Sorry…

Reply:
Woops, not your fault, I've been slow to followups. Moved it over.